src/PeriodEnd/LevyPayments/DeployLevyPayments/sql/dml/05 PeriodEnd.Populate.Reference.PaymentsHistory.dml.sql

```diff
@@ -17,3 +17,13 @@ SELECT
     WHERE rp.Ukprn IN (SELECT DISTINCT [Ukprn] FROM [Reference].[Providers])
```

Comment:
at the top of the table is a TRUNCATE. Need to have a check for the IX_LevyPaymentHistory_RequiredPaymentId index and drop it if it exists.